floyd/cli/data.py

```diff
@@ -141,6 +143,71 @@ def clone(id):
                               delete_after_untar=True)
 
 
+@click.command()
+@click.argument('dataset_name', nargs=1)
+def listfiles(dataset_name):
```

Comment:
if job output is also supported, then `dataset_name` is not accurate anymore, maybe change to `data_name` or just `name`.